xbmc/network/GUIDialogNetworkSetup.cpp

```diff
@@ -131,7 +142,8 @@ bool CGUIDialogNetworkSetup::ShowAndGetNetworkAddress(std::string &path)
   CGUIDialogNetworkSetup *dialog = g_windowManager.GetWindow<CGUIDialogNetworkSetup>(WINDOW_DIALOG_NETWORK_SETUP);
   if (!dialog) return false;
   dialog->Initialize();
-  dialog->SetPath(path);
+  if (!dialog->SetPath(path))
+    HELPERS::ShowOKDialogText(CVariant{ 10218 }, CVariant{ 39103 });
   dialog->Open();
```

Comment:
should we prevent opening the network dialog?

Reply:
yeh that's better. changed.